include/temoto_robot_manager/robot_manager.h

```diff
@@ -31,6 +31,9 @@
 #include "temoto_robot_manager/robot_config.h"
 
 #include <moveit/move_group_interface/move_group_interface.h>
+#include <move_base_msgs/MoveBaseAction.h>
```

Comment:
It is included in the robot_manager due I created a typedef for an action client at the end: 
typedef actionlib::SimpleActionClient<move_base_msgs::MoveBaseAction> MoveBaseClient;
In that case, it would be better to remove it from robot.h than robot_manager.h.